pkg/controller/pravega/bookie.go

```diff
@@ -202,9 +208,19 @@ func MakeBookieConfigMap(pravegaCluster *v1alpha1.PravegaCluster) *corev1.Config
 		"-XX:-ResizePLAB",
 	}
 
+	gcLoggingOpts := []string{
+		"-XX:+PrintGCDetails",
```

Comment:
Probably better to add a timestamp  `-XX:+PrintGCDateStamps`  and a log dir, e.g. `-Xloggc:/tmp/gclog.log`, see [this](https://dzone.com/articles/enabling-and-analysing-the-garbage-collection-log). what do you think?

Reply:
Yes, I'll include the `-XX:+PrintGCDateStamps` option. About logging to a file, I don't think we should do that in Kubernetes, because the root file system is ephemeral and we won't be able to access the log file if the container dies. Instead, we want GC details to be printed to stdout so that it's handled by the logging agent (e.g. fluentd), which is the default behavior.

Reply:
Oh, so the default output is stdout. That makes more sense.